package.json

```diff
@@ -56,7 +56,7 @@
     "buffer-xor": "^2.0.2",
     "chai": "^4.3.1",
     "copyfiles": "^2.3.0",
-    "ethereum-waffle": "3.0.0",
+    "ethereum-waffle": "^3.3.0",
```

Comment:
I see that this version was pinned and is now floating. 

@smartcontracts the blame says this was done in the initial commit. I'm curious if there's a reason the version was previously pinned?

Reply:
Nope. Probably my mistake.